cachito/web/config.py

```diff
@@ -25,6 +26,8 @@ class Config(object):
     # Users that are allowed to use the "user" property when creating a request
     CACHITO_USER_REPRESENTATIVES: List[str] = []
     CACHITO_WORKER_USERNAMES: List[str] = []
+    LOGIN_DISABLED = False
```

Comment:
We can remove this config from the TestConfig class, since it became redundant. (L71)